int64 num_customers
---
string customer
string drink